borg/archiver.py

```diff
@@ -564,7 +569,8 @@ def build_parser(self, args=None, prog=None):
         common_parser.add_argument('--remote-path', dest='remote_path', default=RemoteRepository.remote_path, metavar='PATH',
                                    help='set remote path to executable (default: "%(default)s")')
 
-        parser = argparse.ArgumentParser(prog=prog, description='Borg %s - Deduplicated Backups' % __version__)
+        parser = argparse.ArgumentParser(prog=prog, description='Borg - Deduplicated Backups')
+        parser.add_argument('-V', '--version', **version_argument)
```

Comment:
i think it should show up in borg [-h], but it is not needed to show up as option for every subcommand, that's rather adding unneeded information there.